Catappult/appcoins-wallet-android
# [feature/ESKL-53] - Add some context in withdraw screen

**What does this PR do?**

Refactors WithdrawFragment, having in mind the new layout provided by Luis and the new architecture followed by the Android team. From the initial implementation (and besides the obvious):
1) There's now a minimum amount to withdraw of 20$. It's pending on the backend side an endpoint to return this value, instead of having this fixed on the error message.
2) User e-mail is stored inside SharedPreferences when a user tries to withdraw, even if the withdraw fails.
3) Tries (or _tries_) to implement some sort of landscape behaviour. 

Also, I've left some of the design for when we are prepared to launch the e-Skills marketing. Let me know if you prefer to have this removed.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] app/src/main/java/com/asfoundation/wallet/eskills_withdraw/WithdrawFragment.kt

**How should this be manually tested?**

1) Try to withdraw without having earned any APPC-c from e-Skills games
2) Play (and win) a e-Skills game, like 2048, and try to withdraw. You can also confirm with the backend team if the withdraw request was successfully received.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ESKL-53](https://aptoide.atlassian.net/browse/ESKL-53)

**Questions:**

N/A



**Code Review Checklist**
